### PR TITLE
samples: ssf_client: support sysbuild and remove sdfw_update service from sample

### PR DESCRIPTION
**Pull request by anhmolt**
* app+rad multi image build
* remove call to sdfw_update service
* Clean up sample.yaml and log by default on default serial

### CODE REVIEW

**Comment by nordicjm** [comment]
multi-image build is child/parent image, this should say sysbuild

**Comment by anhmolt** [comment]
> multi-image build is child/parent image, this should say sysbuild

Thanks! Have updated the title.

**Review by FrancescoSer — APPROVED** [review]

File: samples/sdfw/ssf_client/Kconfig.sysbuild [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config BUILD_ALSO_FOR_RADIO_CORE
+	bool "Build also for the radio CPU"
+
+config RADIO_IMAGE_BOARD
+	string "The board used for cpurad target"
+	depends on BUILD_ALSO_FOR_RADIO_CORE
+	default "nrf54h20dk/nrf54h20/cpurad" if BOARD_NRF54H20DK_NRF54H20_CPUAPP
+	default "nrf9280pdk/nrf9280/cpurad" if BOARD_NRF9280PDK_NRF9280_CPUAPP
```
Comment (nordicjm) [inline samples/sdfw/ssf_client/Kconfig.sysbuild:16]: Rework to work as the sysbuild Kconfig files do i.e. rename to `*_BOARD_TARGET_CPUCLUSTER`, reference `SOC_*` Kconfigs and only have the CPU cluster here then work out the board to use properly in cmake - see https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/Kconfig.netcore#L15 and https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/netcore.cmake

Comment (anhmolt) [reply samples/sdfw/ssf_client/Kconfig.sysbuild]: Thanks. I have updated to work out the radio board target in cmake.

**Review by jonathannilsen — APPROVED** [review]

File: samples/sdfw/ssf_client/Kconfig.sysbuild [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config BUILD_ALSO_FOR_RADIO_CORE
+	bool "Build also for the radio CPU"
```
Comment (nordicjm) [inline samples/sdfw/ssf_client/Kconfig.sysbuild:10]: this should have depends on for what needs it and `default y`

File: samples/sdfw/ssf_client/sample.yaml [hunk 3]
```diff
@@ -22,3 +22,15 @@ tests:
...
+    tags: ci_build ci_samples_sdfw sysbuild
+    sysbuild: true
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+      - nrf9280pdk/nrf9280/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+      - nrf9280pdk/nrf9280/cpuapp
+    extra_args:
+      SB_CONFIG_BUILD_ALSO_FOR_RADIO_CORE=y
```
Comment (nordicjm) [inline samples/sdfw/ssf_client/sample.yaml:36]: and this should not be present

File: samples/sdfw/ssf_client/sysbuild.cmake [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+string(REPLACE "/" ";" split_board_qualifiers "${BOARD_QUALIFIERS}")
+list(GET split_board_qualifiers 1 target_soc)
+list(GET split_board_qualifiers 2 target_cpucluster)
+set(board_target_radiocore "${BOARD}/${target_soc}/${SB_CONFIG_RADIO_BOARD_TARGET_CPUCLUSTER}")
+set(target_soc)
+set(target_cpucluster)
```
Comment (nordicjm) [inline samples/sdfw/ssf_client/sysbuild.cmake:12]: this goes in the if() part below

**Review by FrancescoSer — APPROVED** [review]

**Pull request closed without merging**